config/locales/devise.two_factor_authentication.en.yml

```diff
@@ -9,7 +9,9 @@ en:
         Your account is temporarily locked because you have entered the one-time passcode
         incorrectly too many times.<br /><br />
         Please try again in %{time_remaining}.
-      otp_setup: Every time you log in, you will be sent one-time passcode to your phone via SMS.  Please enter the phone number you would like to use to receive these messages.
+      otp_setup: >
+        Every time you log in, a one-time passcode will be sent to your phone via SMS.
+        Please enter the phone number you would like to use to receive these messages.
```

Comment:
Why this change?   If its just to wrap at 80 cols then the message below should be updated too.  Maybe a separate change?